src/inmanta/agent/executor.py

```diff
@@ -397,14 +394,14 @@ async def clean_environments(self) -> None:
         environments_on_disk_to_clean: set[str] = set()
         reverse_environment_map = {v.env_path: k for k, v in self._environment_map.items()}
 
-        # We check every folder and we save the ones that contain the special file and that should be removed
+        # We check every folder, and we save the ones that contain the special file and that should be removed
         current_datetime = datetime.datetime.now()
         for file in envs_dir.iterdir():
             if not file.is_dir():
                 continue
 
             absolute_path_file = str(file.absolute())
-            inmanta_env_status_file = file / ".inmanta_env_status"
+            inmanta_env_status_file = file / f".{const.INMANTA_ENV_STATUS_FILENAME}"
```

Comment:
the `.` has to be part for the constant as well (as it is part of the file name)